src/main/groovy/com/github/mrsarm/jshell/plugin/JShellPlugin.groovy

```diff
@@ -63,6 +63,10 @@ class JShellPlugin implements Plugin<Project> {
                                        'the following paths from project configurations: {}', depsPaths)
                 pathSet.addAll(depsPaths)
             }
+            
+            // Exclude non-directory / non-jar files that jshell doesn't deal with
+            pathSet = pathSet.findAll{ it.isDirectory() || it.toString().endsWith('.jar') }
```

Comment:
Testing it locally with a sample project, I got this error:

```
$ gradle --console plain jshell
> Task :compileJava UP-TO-DATE
...
> No signature of method: java.lang.String.isDirectory() is applicable for argument types: () values: []
```

Looks like `it` is a `String` object and does not have a `isDirectory()` method.

Reply:
```suggestion
            pathSet = pathSet.findAll{ new File(it.toString()).isDirectory() || it.toString().endsWith('.jar') }
```

Please apply this patch to your PR.